packages/sdk/app-framework/src/plugins/PluginHost/plugin.ts

```diff
@@ -107,8 +109,6 @@ export type PluginDefinition<TProvides = {}, TInitializeProvides = {}> = Omit<Pl
   unload?: () => Promise<void>;
 };
 
-export const pluginMeta = (meta: Plugin['meta']) => meta;
```

Comment:
but it imports the app-framework dependency.

Reply:
that's fair, though it should be tree-shaken to just this function which is negligible, but there is a difference between this and config because these are runtime. I think I'm okay with this api, it feels slightly clunkier to me than a function, but it's nice that it compiles away to nothing.